js/checkbox-wikirule.js

```diff
@@ -79,40 +72,54 @@ exports.parse = function() {
                 this.parser.source.substring(startPos + 4, this.parser.pos),
                 {parseAsInline: true});
 
-        // Put the listitem body in a span for easy reference
-        var itembody = {
-            type: "element",
-            tag: "span",
-            attributes: {
-                class: {type: "string", value: "checklistitem-body"}
```

Comment:
if checklistitem-body is no longer used it should be removed as part of this commit. Each commit should be a functional intermediate state.